app-portage/emlop/emlop-0.7.0.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.5.4
```

Comment:
We recommend using pycargoebuild at this point (as it won't generate output causing warnings, for a start).

Reply:
Looks like pycargoebuild doesn't generate

    # rust does not use *FLAGS from make.conf, silence portage warning                                                                                                                                                                          
    QA_FLAGS_IGNORED="usr/bin/${PN}"

TBH I don't fully understand what it does, so I left it in for now. Should I remove it ?

Reply:
Creating the right value for QA_FLAGS_IGNORED is non-trival for tools. And it is still required to silence the QA warning, so we want to keep it.